microscopium/serve.py

```diff
@@ -267,6 +270,19 @@ def update_table(indices, df, table):
     filtered_df = df.iloc[indices]
     table.source.data = ColumnDataSource(filtered_df).data
 
+def switch_modes_button_group():
+    from bokeh.models.widgets import RadioButtonGroup
```

Comment:
Imports go at the top of the file - see PEP8 https://legacy.python.org/dev/peps/pep-0008/#imports